drivers/wireless/gs2200m.c

```diff
@@ -2152,6 +2152,35 @@ static enum pkt_type_e gs2200m_activate_wrx(FAR struct gs2200m_dev_s *dev,
   return gs2200m_send_cmd2(dev, cmd);
 }
 
+/****************************************************************************
+ * Name: gs2200m_powersave_wrx
+ * NOTE: See 9.1.1 Active Radio Receive
+ ****************************************************************************/
+
+static enum pkt_type_e gs2200m_powersave_wrx(FAR struct gs2200m_dev_s *dev,
+                                            uint8_t on)
```

Comment:
add space after uint8_t

Reply:
@xiaoxiang781216 
Thank you for reviewing.
I add it.
Please check it.